### ISSUE TITLE
Make gamification and achievement notifications optional

### ISSUE THREAD

**Issue by bxl-forever** [westnordost/StreetComplete#2132]
Achievements are a nice feature but they are not useful for everyone.

Some of us are experienced OSM users, who use StreetComplete when out on the field.  Being interrupted to be praised for helping OSM or for our latest achievement or streak is merely an annoyance.

I suggest to create an option in the Settings, with a check box making it possible to disable those.

**Comment by westnordost** [comment]
I want to keep the amount of settings to a minimum actually. Having an option like this falls into the same category as "remove all those wonky animations" etc., I am not convinced that this is really so important that it needs an option to be turned off.

Since the redesign, the achievement notification should also not pop up directly but wait in the notification button to be shown, is this not working as it should?

**Comment by rhhsm** [comment]
Speaking of annoyances, I find it much more annoying that after being a regular contributor for some time, I still have to confirm twice at the start of each session that I'll only add data based on fieldwork. Maybe it can be made a reward that these disappear after scoring 1000 points? :)

**Comment by peternewman** [comment]
> Speaking of annoyances, I find it much more annoying that after being a regular contributor for some time, I still have to confirm twice at the start of each session that I'll only add data based on fieldwork. Maybe it can be made a reward that these disappear after scoring 1000 points? :)

I don't! I do have a few more than 1000, but I doubt it's set higher than that. I just get warned if I'm not near enough to the place.

**Comment by smichel17** [comment]
Currently, the "Do not show this dialog for the rest of the session" checkbox only appears in the *second* confirmation box. Perhaps at 1000 quests it could start showing in the first confirmation dialog?

I will add, as I have completed more quests (currently just over 2000), I have appreciated more the value of on-site surveys / the difference between "almost all of the roads around here are asphalt, and I'm pretty sure this one is too" compared to verifying it on-site. So I think it is good that the app is annoying about this near the beginning (and perhaps could stand to be even more annoying), but after hundreds or thousands of answers, if one does not understand the value of on-site surveying, a few extra boxes to click through is not going to stop them from remotely surveying, so there's not much of a reason to stay that annoying.
